docs/DeveloperGuide.adoc

```diff
@@ -295,72 +319,395 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+|`* * *` |user |find a transaction by description |locate details of transactions without having to go through the entire list
+
+|`* * *` |user |sort transactions by date, amount and alphabetical order of name of person who spent the amount |view and prioritise reimbursements of transactions
+
+|`* * *` |user |an overview of the expenditure for each event| that I can keep track for future purposes or reporting to the school
+
+|`* * *` |user |track individual sales at an event and tabulate total sales at the end of the day |
+
+|`* * *` |user |find transactions by a single person and if reimbursment has been done|locate total amount of money to reimburse the person and keep track of reimbursements
+
+|`* * *` |user |function to schedule goals and plan budget/spending |
+
+|`* * *` |user |have information of the person I need to reimburse| easily find information to contact the person for reimbursement
+
+|`* * *` |user|generate and export to print out overview reports of the financials| present it during board meetings and give it to other board directors
+
+|`* * *` |user |have a cashier mode for another member who do sales input data directly in real time when the transaction is made | not spend extra time to collate their sales and revenue and key it in myself
+
+|`* * *` |user |plan and estimate my budget for events by comparing with previous transactions|
+
+|`* *` |user |can tag and see who I have to reimburse back to most urgently |
+
+|`* *` |user |have a reminder pop up when I open the app of the reimbursements I have to do within this week from the current date |
+
+|`*` |user |schedule deadlines in order to plan for events |
+
+|`*` |user|auto complete for people already in database |know if I have to get their contact details to fill into the database later
+
+|`*` |user |upload receipt proofs into the system| better check for validity of transaction and for security
+
+|`*`|user |upload receipt proofs into the system| better check for validity of transaction and for security
 
-|`*` |user with many persons in the address book |sort persons by name |locate a person easily
 |=======================================================================
 
-_{More to be added}_
 
 [appendix]
 == Use Cases
 
-(For all use cases below, the *System* is the `AddressBook` and the *Actor* is the `user`, unless specified otherwise)
+(For all use cases below, the *System* is the `treasurerPro (tP)` and the *Actor* is the `user`, unless specified otherwise)
+
+=== Overview Tab
 
 [discrete]
-=== Use case: Delete person
+=== Use Case 1: Sets goals for expenditure, budget and sales
+
+*Guarantees*
+
+- Financial goals are only valid if they are a positive, non-zero amount
+- Financial goals are aligned within a time period
 
 *MSS*
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1. The user arrives on the Overview tab
```

Comment:
Is the command format correct? Shouldn't all the information be entered in a one single step? Is there any system response?